src/player.h

```diff
@@ -148,37 +148,39 @@ namespace Player {
 	 */
 	std::string GetEncoding();
 
-	/**
-	 * @return Whether engine is RPG2k
-	 */
+	/** @return If engine is any version of RPG2k */
 	bool IsRPG2k();
 
-	/**
-	 * @return If engine is RPG2k3 v1.09a or older
-	 */
-	bool IsRPG2k3Legacy();
-
-	/**
-	 * @return If engine is RPG2k3
-	 */
+	/** @return If engine is any version of RPG2k3 */
 	bool IsRPG2k3();
 
-	/**
-	 * @return If engine is RPG2k v1.50 or newer, or RPG2k3 v1.05 or newer
-	 */
-	bool IsMajorUpdatedVersion();
+	/** @return If engine is RPG2k <= 1.10 */
+	bool IsRPG2kLegacy();
 
-	/**
-	 * @return If engine is the official English RM2k3 release (v1.10) or newer.
-	 */
-	bool IsRPG2k3E();
+	/** @return If engine is RPG2k3 <= v1.04 */
+	bool IsRPG2k3Legacy();
+
+	/** @return If engine is RPG2k >= 1.50 and RPG2k <= 1.52 */
+	bool IsRPG2kUpdated();
+
+	/** @return If engine is RPG2k3 >= 1.05 and RPG2k3 <= 1.09a */
+	bool IsRPG2k3Updated();
 
 	/**
 	 * @return If engine is the official English RM2k release v.1.61 or newer.
 	 * False if engine is RM2k3, Japanese, unofficial or v.1.60.
 	 */
 	bool IsRPG2kE();
 
+	/** @return If engine is the official English RM2k3 release (v1.10) or newer. */
+	bool IsRPG2k3E();
+
+	/** @return If engine is RPG2kLegacy() or RPG2k3Legacy() */
+	bool IsLegacy();
+
+	/** @return If engine is RPG2kUpdated() or RPG2k3Updated() */
+	bool IsMajorUpdatedVersion();
```

Comment:
You changed the range of the functions, e.g. before "IsMajorUpdated" was "Everything not Legacy" and now it is "Every not Legacy and not English/Steam" or are the comments misleading?
When this is intended we need a better naming here. Maybe split between 2kE and 2kJ and for J add additional split 2kJLegacy and 2kJUpdated?

Reply:
I can change the comments, updated should just be `>=`

We do need better naming. And maybe even finer grained control. Because there are major features differences between these lines. For this PR I just wanted to clean it up a little and support legacy rm2k3, but not completely redo it as I am not yet sure what it should look like.